docs/standard-library/in-place-t-struct-in-place-type-t-struct-in-place-index-t-struct.md

```diff
@@ -0,0 +1,36 @@
+---
+description: "Learn more about: in_place_t Struct, in_place_type_t Struct, in_place_index_t Struct"
+title: "in_place_t Struct, in_place_type_t Struct, in_place_index_t Struct"
+ms.date: "11/04/2016"
+f1_keywords: ["utility<in_place_t>", "utility/std::in_place_t", "utility<in_place_type_t>", "utility/std::in_place_type_t", "utility<in_place_index_t>", "utility/std::in_place_index_t"]
+helpviewer_keywords: ["utility<in_place_t> struct"]
+---
+# in_place_t Struct, in_place_type_t Struct, in_place_index_t Struct
+
+The empty structure type `in_place_t` can be passed into constructors of `expected` class, [`optional` class](optional-class.md) or [`single_view` class](single-view-class.md) for in-place construction of the contained type. The empty structure type `in_place_type_t` can be passed into constructors of [`any` class](any-class.md) or [`variant` class](variant-class.md) to specify the type of the object. The empty structure type `in_place_index_t` can be passed into constructors of [`variant` class](variant-class.md) to specify the index of the object.
```

Comment:
Weren't these introduced in C++17? We should tag them as such and indicate that they should use at least **`/std:c++17`** to compile.